src/mappers/business-address-lookup-mapper.js

```diff
@@ -0,0 +1,63 @@
+/**
+ * Maps addresses returned by the OS Places API into a format suitable
+ * for front-end display and DAL updating.
+ * https://docs.os.uk/os-apis/accessing-os-apis/os-places-api/datasets as a reference for the datasets
+ *
+ * @module businessAddressLookupMapper
+ */
+
+import { COUNTRY_NAMES } from '../constants/country-names.js'
+
+const businessAddressLookupMapper = (addresses) => {
+  return addresses.map((address) => {
+    const {
+      UPRN, // always present
+      ADDRESS, // always present
+      ORGANISATION_NAME, // optional
+      DEPARTMENT_NAME, // optional
+      SUB_BUILDING_NAME, // optional
+      BUILDING_NAME, // optional
+      BUILDING_NUMBER, // optional
+      DEPENDENT_THOROUGHFARE_NAME, // optional
+      THOROUGHFARE_NAME, // optional
+      DOUBLE_DEPENDENT_LOCALITY, // optional
+      DEPENDENT_LOCALITY, // optional
+      POST_TOWN, // always present
+      POSTCODE, // always present
+      LOCAL_CUSTODIAN_CODE_DESCRIPTION, // always present
+      COUNTRY_CODE // always present
+    } = address.properties
+
```

Comment:
Instead of having the properties of the returned OPS fields as inline comments within the code, why not create a schema for the set of the returned fields and then validate all data received against the schema.